orienteer-core/src/main/resources/org/orienteer/core/web/orienteer-coreui.css

```diff
@@ -63,7 +77,7 @@ li.strict caption {
 }
 
 .btn-xs {
-  line-height: .38rem;
+  line-height: .76rem;
```

Comment:
Why separate CSS instead of adjusting in orienteer.css?

Reply:
You not approve it with Vitaly? Lets discuss